### PR TITLE
Node server refactor

### PR DESCRIPTION
**Pull request by lwaghorn**

### CODE REVIEW

File: OnLand/ControlLaptop/templates/_cards/cells1.html [hunk 1]
```diff
@@ -7,294 +7,14 @@
                 <th>Voltage</th>
                 <th>Temp</th>
             </tr>
-            <tr>
+            {% for i in range(20) %}
+            <tr id="cell-{{ i }}">
```
Comment (andrewnash) [inline OnLand/ControlLaptop/templates/_cards/cells1.html:11]: I have no idea why I didn't do this

Comment (andrewnash) [reply OnLand/ControlLaptop/templates/_cards/cells1.html:11]: Please take a moment to appreciate how stupid I was

Comment (lwaghorn) [reply OnLand/ControlLaptop/templates/_cards/cells1.html:11]: Noone thinks straight that close to finals

File: OnPod/FlightComputer/examples/nodeSim.cpp [hunk 2]
```diff
@@ -12,45 +12,40 @@
...
-	std::string sPayload = "";
-	 int32_t iProtoPacketSize = pNodeUpdate.ByteSizeLong();
-	pNodeUpdate.SerializeToString(&sPayload);
-	void* vBuffer = malloc(iProtoPacketSize);
-	pNodeUpdate.SerializeToArray(vBuffer, iProtoPacketSize);
-	sendDataUdp(cscSocket, vBuffer, (int32_t)iProtoPacketSize);
+void sendPacket(brakeNodeData pNodeUpdate, clientSocketConfig *cscSocket) {
+    std::string sPayload = "";
+    int32_t iProtoPacketSize = pNodeUpdate.ByteSizeLong();
+    pNodeUpdate.SerializeToString(&sPayload);
```
Comment (lwaghorn) [inline OnPod/FlightComputer/examples/nodeSim.cpp:18]: sPayload literally does nothing?

**Pull request merged**